### ISSUE TITLE
android: home scrollers do not always scroll all the way up

### ISSUE THREAD

**Issue by Brahmanathaswami** [Himalayan-Academy/Siva-Siva-App#150]

**Comment by Brahmanathaswami** [comment]
Thank you for your request.

I have been able to recreate the issue where the home portal scroller will not
scroll to the bottom of the list, it seems like it might be to do with sending
the CreateScroller command in 100 millisecond and the wait for 100
milliseconds. I changed the code to

on openCard
   portal_SetCurrentStack "Siva-Siva-Portal"
   portal_SetCurrent (the short name of this stack)

   put "SivaSiva Portal Scrolling" into tLinksGroup
   set the acceleratedRendering of this stack to true
  
   send "CreateScroller tLinksGroup" to grp "SivaSiva Portal Scrolling"
end openCard

and it seems to work. It may be that the send in time and wait are there for a
reason in which case this solution may not work.

I have not seen a crash though.

I also tested the puzzle and did not get a crash, have you been able to
recreate this these issues yourself?

Kind regards

Elanor
